extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -403,10 +342,10 @@ public function combinedMediaSearch() {
 		$config = new Wikia\Search\Config;
 		$videoOnly = (bool) $request->getVal( 'videoOnly', false );
 		$config->setQuery( $query )
-		       ->setCombinedMediaSearch( true )
-		       ->setCombinedMediaSearchIsVideoOnly( $videoOnly )
-		       ->setLimit( 4 )
-		       ->setStart( $this->getVal( 'next', 0 ) );
+			->setCombinedMediaSearch( true )
+			->setCombinedMediaSearchIsVideoOnly( $videoOnly )
+			->setLimit( 4 )
+			->setStart( $this->getVal( 'next', 0 ) );
 
 		$results = $this->queryServiceFactory->getFromConfig( $config )->searchAsApi( [ 'url', 'id', 'pageid', 'wid', 'title' ], true );
 		$dimensions = [ 'width' => 120, 'height' => 90 ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "height" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "width" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)